src/machine/machine_nrf.go

```diff
@@ -20,38 +20,38 @@ const (
 func (p GPIO) Configure(config GPIOConfig) {
 	cfg := config.Mode | nrf.GPIO_PIN_CNF_DRIVE_S0S1 | nrf.GPIO_PIN_CNF_SENSE_Disabled
 	port, pin := p.getPortPin()
-	port.PIN_CNF[pin] = nrf.RegValue(cfg)
+	port.PIN_CNF[pin].Set(uint32(cfg))
 }
 
 // Set the pin to high or low.
 // Warning: only use this on an output pin!
 func (p GPIO) Set(high bool) {
 	port, pin := p.getPortPin()
 	if high {
-		port.OUTSET = 1 << pin
+		port.OUTSET.Set(1 << pin)
 	} else {
-		port.OUTCLR = 1 << pin
+		port.OUTCLR.Set(1 << pin)
 	}
 }
 
 // Return the register and mask to enable a given GPIO pin. This can be used to
 // implement bit-banged drivers.
 func (p GPIO) PortMaskSet() (*uint32, uint32) {
 	port, pin := p.getPortPin()
-	return (*uint32)(&port.OUTSET), 1 << pin
+	return (*uint32)(&port.OUTSET.Reg), 1 << pin
```

Comment:
This cast (and the one below) is unnecessary.

Reply:
OK fixed.